uniflow/flow/flow_factory.py

```diff
@@ -1,10 +1,18 @@
 """Flow Factory Module."""
+from typing import Dict, List
+
+from uniflow.constants import BASIC, EXTRACT, MODEL, TRANSFORM
 
 
 class FlowFactory:
     """Flow Factory Class."""
 
-    _flows = {}
+    _flows = {
+        BASIC: {},
+        EXTRACT: {},
+        MODEL: {},
+        TRANSFORM: {},
```

Comment:
you factory should only contain extract and transform flows. basic and model should not be here.